fbpic/lpa_utils/bunch.py

```diff
@@ -405,8 +387,8 @@ def add_elec_bunch_openPMD( sim, ts_path, z_off=0., species=None, select=None,
 
     # Transform particle distribution in
     # the Lorentz boosted frame, if gamma_boost != 1.
```

Comment:
I also corrected the lines below: it seems to be a previous error, since the method `boost_particles` does not return anything.